package.json

```diff
@@ -1,9 +1,8 @@
 {
 	"name": "skinview3d",
-	"version": "2.0.0-alpha.5",
+	"version": "2.0.1",
 	"description": "Three.js powered Minecraft skin viewer",
 	"main": "libs/skinview3d.js",
-	"type": "module",
```

Comment:
@yushijinhun @g-plane storybook was not a fan of this. Does removing it cause any concern or issues?

Reply:
Is removing this necessary?

Reply:
Storybook throws this error if I don't remove it.

```log
WARN Unable to close preview build!
ERR! Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: D:\Code\Gaming\skinview3d\.storybook\main.js
ERR! require() of ES modules is not supported.
ERR! require() of D:\Code\Gaming\skinview3d\.storybook\main.js from D:\Code\Gaming\skinview3d\node_modules\@storybook\core\dist\server\utils\server-require.js is an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which defines all .js files in that package scope as ES modules.
ERR! Instead rename main.js to end in .cjs, change the requiring code to use import(), or remove "type": "module" from D:\Code\Gaming\skinview3d\package.json.
```

Reply:
Resolved in https://github.com/bs-community/skinview3d/commit/86731e23b9b1cc8f4855471f179080a1bdd42843